cygnus-common/src/main/java/com/telefonica/iot/cygnus/utils/CommonConstants.java

```diff
@@ -68,6 +68,7 @@ private CommonConstants() {
     public static final String LOG4J_COMP = "agent";
     
     // encoding
+    public static final String INTERNAL_CONCATENATOR = "=";
```

Comment:
Jejeje I was waiting for this :) This is because there is a first stage were a part of the table name is composed. I firstly tried to use the `x0000` concatenator for this part building, but then I realized that, when building the final table, I was encoding the `x0000` as `xx0000`. Thus, I cherry-picked a trick from Orion xD : I use the character `=` (which is forbiden by Orion, thus it will never be notified) as an internal concatenator character that is especially encoded as `x0000`.